fast_response/listeners/gcn_listener.py

```diff
@@ -101,17 +101,23 @@ def process_gcn(payload, root):
     if args.run_live:
         print("Listening for GCNs . . . ")
         gcn.listen(handler=process_gcn)
-    elif not args.test_cascade:
-        print("Running on sample track . . . ")
-        payload = open('/data/user/apizzuto/fast_response_skylab/' \
-            + 'fast-response/fast_response/sample_skymaps/' \
-            + 'sample_astrotrack_alert_2021.xml', 'rb').read()
-        root = lxml.etree.fromstring(payload)
-        process_gcn(payload, root)
     else:
-        print("Running on sample cascade . . . ")
-        payload = open('/data/user/apizzuto/fast_response_skylab/' \
-            + 'fast-response/fast_response/sample_skymaps/' \
-            + 'sample_cascade.txt', 'rb').read()
-        root = lxml.etree.fromstring(payload)
-        process_gcn(payload, root)
+        try:
+            import fast_response
+            sample_skymap_path=os.path.dirname(fast_response.__file__) +'/sample_skymaps/'
+        except Exception as e:
+            sample_skymap_path='/data/user/apizzuto/fast_response_skylab/' \
```

Comment:
Might be worth moving these to where the fast response scripts live in `/data/ana/` so we aren't relying on my hardcoded path, but this is totally fine to leave as is for now

Reply:
I was trying to leave this as a backup to the one that it should be finding in fast response, but maybe I'll have it just print an error instead and remove this

Reply:
Fixed in https://github.com/icecube/FastResponseAnalysis/pull/24/commits/cc053cc7feebeec862207d55509ea6aa4875764c